app/models/post.rb

```diff
@@ -1,3 +1,19 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  after_create :update_user_posts_counter
+
+  def update_user_posts_counter
+    author&.update(posts_counter: author.posts.count)
+  end
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
```

Comment:
- [ ] To meet the below requirement please add unit tests for this custom method created for the post model :+1: 
![mm](https://github.com/NoelLincoln/blog-app/assets/89055502/1a7065fb-a00b-41cf-b3e5-3f72e0e2a85e)